packages/datetime/test/dateRangePickerTests.tsx

```diff
@@ -406,8 +406,10 @@ describe("<DateRangePicker>", () => {
 
             renderDateRangePicker({ contiguousCalendarMonths, maxDate, minDate });
             assert.lengthOf(document.getElementsByClassName("DayPicker"), 1);
-            assert.lengthOf(document.getElementsByClassName("DayPicker-NavButton--prev"), 0);
-            assert.lengthOf(document.getElementsByClassName("DayPicker-NavButton--next"), 0);
+            // react-day-picker still renders the navigation but with a interaction disabled class
+            assert.lengthOf(document.getElementsByClassName("DayPicker-NavButton--prev"), 1);
+            assert.lengthOf(document.getElementsByClassName("DayPicker-NavButton--next"), 1);
```

Comment:
Not sure if there is a better way to actually assert visibility of these elements, given they are now always rendered.

Reply:
Do we need to reach into the `style` of each element to check the `visibility`?

Reply:
what about looking for that conditional class?

Reply:
or, if they're always there, just remove these assertions. no need to assert on external library behavior. case in point: when that behavior changes and breaks your tests (but our code is still fine).